classes/asserter.php

```diff
@@ -136,13 +136,13 @@ protected function getTypeOf($mixed)
         return $this->analyzer->getTypeOf($mixed);
     }
 
-    protected function _($string)
+    protected function _(...$arguments)
     {
-        return call_user_func_array([$this->locale, '_'], func_get_args());
+        return call_user_func_array([$this->locale, '_'], $arguments);
     }
 
-    protected function __($singular, $plural, $quantity)
+    protected function __(...$arguments)
```

Comment:
Would be good to keep the `singular`, `plural`, and `quantity` somewhere, because `$arguments` has no special meaning :-/.